src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,90 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "Orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
```

Comment:
Consider adding the `mappedBy` attribute to the `@OneToMany` annotation to specify the owning side of the relationship. For example, `@OneToMany(mappedBy = "order")` if the `Ticket` entity has a field named `order`.